spec/peg_zone_spec.md

```diff
@@ -99,11 +99,11 @@ to which the Cosmos peg zone is listening.
 
 ![Cosmos to Ethereum](./pegzone-to-ether.jpg)
 
-1. the ABCI app receives an IBC packet from the hub that requests for locking Cosmos tokens and handles it according to the IBC specification. The ABCI app generates a valid Ethereum transaction containing {address, denomination, amount, nonce}, and writes it to its state. 
+1. the ABCI app receives an `IBCOutgoing` from the hub that requests for locking Cosmos tokens and handles it according to the IBC specification. The ABCI app generates a valid Ethereum transaction containing {address, denomination, amount, nonce}, and writes it to its state. 
```

Comment:
Actually all these are `IBCMsg`. They are just posted into `IncomingQueue` and `OutgoingQueue`.

Reply:
The naming I used here is bit ambiguous. In this api, all 'Incoming's means that is coming from the Ethereum contract, and all 'Outgoing's means that is going to the Ethereum contract. It confuses with IBC perspective, so changing them to 'Witness' and 'Signature' will solve this situation.